content/services/development.md

```diff
@@ -8,10 +8,151 @@ image:
 action:
   text: See work samples »
   url: /work/
+promise:
+  - title: Accessible
+    icon: icons/big-circle
+    text: |
+      *The world is diverse, and no person is an edge case.*
+  - title: Approachable
+    icon: icons/big-circle
+    text: |
+      *Your friend in digital design, development, and performance*
+  - title: Human
+    icon: icons/big-circle
+    text: |
+      *Great software requires a personal touch.*
+  - title: Nimble
+    icon: icons/big-circle
+    text: |
+      *Our tightly-integrated team can adapt quickly to your needs*
+  - title: Queer
+    icon: icons/big-circle
+    text: |
+      *Moving past assumptions of “normal” to explore new possibilities.*
+  - title: Robust
+    icon: icons/big-circle
+    text: |
+      *100% test coverage and documentation keep maintenance costs low*
+  - title: Thoughtful
+    icon: icons/big-circle
+    text: |
+      *We don’t have all the answers, but we know how to get there.*
+  - title: Top-shelf
+    icon: icons/big-circle
+    text: |
+      *From CSS/Sass & Django to Susy & Herman, we build the tools that millions of developers rely on.*
+  - title: Transparent
+    icon: icons/big-circle
+    text: |
+      *Daily communication keeps you in control of budget & priorities from start to finish.*
+process:
+  - title: Lean & Agile
+    text: |
+      We use a lean and agile process to deploy & integrate new functionality frequently, aiming to deliver usable product launches as early and often as possible.
+  - title: Transparent & Interactive
+    text: |
+      We encourage clients to stay closely connected with the daily progress, offering frequent feedback and re-prioritization via Trello or Pivotal Tracker.
+  - title: Accessible & Responsive
+    text: |
+      We use the accessible open web stack: semantic HTML5, unobtrusive JavaScript, and advanced CSS – backed by Python & Django on the server.
 summary: |
-  **We integrate design and development in an agile process**,
-  so you only need to hire one team --
-  completely focused on achieving your goals
-  with smooth and efficient progress
-  from concept to launch and beyond.
+  **We integrate design and development in an agile process**, so you only need to hire one team – completely focused on achieving your goals with smooth and efficient progress from concept to launch and beyond. Starting from user needs, we collaborate with you to deliver usable features quickly and efficiently.
+
+  We provide expertise across a range of technologies, including Python/Django, accessible HTML, modern Sass/CSS, Vue, Node, Backbone/Marionette, and more.
 ---
+
+{% import 'embed.macros.njk' as embed %}
+{% import 'quotes.macros.njk' as quotes %}
+{% import 'layout.macros.njk' as layout %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['investment', 'curious']
+) }}
```

Comment:
@mirisuzanne the Emmy's Workshop quote is on a page that is excluded from collections. Is there a good way around this to show that quote on this page?

if not, @SondraE is there a different quote we could use on this page instead of the "realistic" one that's shown on page 9 of your XD file: https://xd.adobe.com/view/b8f11a7c-c6a3-4524-410f-d08c6df679b3-cec0/screen/bb6add8a-7784-40d2-b82b-e649fa02174c/Service-Tag-Page